static/js/lib/syntaxhighlighter/shCore.js

```diff
@@ -2633,6 +2633,64 @@ sh.Highlighter.prototype = {
 		return matches;
 	},
 
+    /* Combines functionality of old findMatches(), getMatches() and removeNestedMatches()
+     * functions. Parse a string of code using an array of regular expressions.
+     * http://jmrware.com/articles/2011/sh-bug/SyntaxHighlighterBug.html
+     *
+     * @param {Array} regexList     List of regular expressions.
+     * @param {String} code         Source code.
+     * @return {Array}              Returns list of matches.
+     */
+    findMatchesNew: function(regexList, code) { // Rev:20110216_1700
+        var matches = []; // Return value. Array of Match objects.
```

Comment:
This has been taken from http://jmrware.com/articles/2011/sh-bug/SyntaxHighlighterBug.html via https://github.com/SyntaxHighlighter/SyntaxHighlighter/issues/72